### PR TITLE
public dashboards release note 

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-06-13-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - dashboard
+  - public 
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Our new **Public Dashboards** feature enables you to share your dashboards with users who do not have a Sumo Logic account. With this feature enabled, you are not allowed to share your dashboard outside the organization with the User's data access level, and to view the shared dashboard, users always need to enable the auto-refresh.
+
+Check out our technical documentation [here](/docs/dashboards-new/share-dashboard-outside-org) to learn more about public dashboards.
```
Comment (kimsauce) [inline blog-service/2023-06-13-dashboards.md:15]: broken link

Comment (JV0812) [reply blog-service/2023-06-13-dashboards.md:15]: Yeah, the doc referred here is not yet published yet. Will be published this week.

**Comment by jpipkin1** [comment]
@JV0812, when will this feature be released? You'll need to change the date on the release note file name to the release date.

**Comment by JV0812** [comment]
Hey John, still is the feature is not up. I will change the date while releasing the doc.

**Comment by jpipkin1** [comment]
The release date is tentatively set for July 11. I changed the date on the file name.

**Review by jpipkin1 — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

**Pull request merged**